### PR TITLE
Bump FPU to pulp-v0.1.1

### PR DESCRIPTION
**Pull request by lucabertaccini**
Bump FPU to `pulp-v0.1.1` which supports:

- Low and mixed-precision SDOTP operations (with stochastic rounding support)
- Compressed vector compare results
- `FP8alt (1, 4, 3)` format

CVA6 does not support the features listed above which are thus disabled in the CVA6 FPU instance.

### CODE REVIEW

**Comment by niwis** [comment]
Thanks a lot, LGTM. I have just two general questions:
1. Will these changes be upstreamed to `openhwgroup:master` in the near future, or is this a more permanent solution?
2. Can you confirm that adding OpenE906 is necessary for this PR? Maybe we need to double-check the license implications for this repo.

**Comment by lucabertaccini** [comment]
Hi @niwis:
1. The integration into `openhwgroup:master` has not been discussed yet
2. The DivSqrt unit from OpenE906 is not relevant for this PR (it is supported only by FP32-only CVFPU instances), but it is aligned with recent OpenHW developments (`develop` branch https://github.com/openhwgroup/cvfpu). In CVFPU, two license-related files have been added during the integration of this alternative DivSqrt module (https://github.com/openhwgroup/cvfpu/blob/develop/LICENSE.apache and https://github.com/openhwgroup/cvfpu/blob/develop/README.license.md)

**Comment by niwis** [comment]
@lucabertaccini 
1. Okay, in that case I think we should make sure to regularly update the `pulp-platform:pulp` branch to include any relevant changes from `openhwgroup:master` so that they can be easily integrated into pulp-CVA6 (and other depandants)
2. would it be an option then to exclude the `vendor` directory in `openhwgroup_cvfpu.vendor.hjson`? To avoid unnecessary inflation of the code base (see [this example](https://github.com/pulp-platform/snitch/blob/9c3fd2bffe6cb88db24bcc7162e46d3fc27ffad0/hw/vendor/openhwgroup_cva6.vendor.hjson#L15))

File: vendor/openhwgroup/cvfpu/util/vendor.py
Comment (niwis) [inline vendor/openhwgroup/cvfpu/util/vendor.py:1]: I think we do not need to add vendor again... I will go ahead and remove the `util` dir

**Review by niwis — APPROVED** [review]

**Pull request merged**